### PR TITLE
resolves #177 allow to override the built-in slim templates

### LINKED ISSUE #177: --template-dir argument doesn't seem to override the built-in slim templates
I'd like to use the `--template-dir` asciidoctor argument to change formatting produced by the default asciidoctor-revealjs slim templates. I've used this successfully with asciidoctor-bespoke, but when doing the same with asciidoctor-revealjs it doesn't seem to work (the backend template is not overridden). I'm thinking this might be due to #131 but would like to know if `-T` is supported or if there's a workaround that doesn't involve tinkering with the asciidoctor-revealjs backend.

thanks,

### PR DESCRIPTION
**Pull request by Mogztter**
resolves #177

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -94,6 +96,11 @@ DocTest::RakeTasks.new do |t|
   t.converter_opts = { backend_name: 'revealjs' }
 end
 
+Rake::TestTask.new(:test) do |t|
```
Comment (Mogztter) [inline Rakefile]: I've created a test task to run tests in the `test/asciidoctor-revealjs` directory.
Previously, the existing file `test/asciidoctor-revealjs/version_test.rb` was not executed.

File: test/test_helper.rb [hunk 2]
```diff
@@ -7,5 +7,4 @@
 
 require 'asciidoctor/doctest'
 require 'minitest/autorun'
-require 'minitest/rg'
```
Comment (Mogztter) [inline test/test_helper.rb]: This dependency is not explicitly declared (ie. missing)

File: Rakefile [hunk 3]
```diff
@@ -132,7 +139,8 @@ namespace :examples do
   end
 end
 
+task 'test' => 'doctest'
 task 'doctest:test' => 'prepare-converter'
 task 'doctest:generate' => 'prepare-converter'
 # When no task specified, run test.
-task :default => :doctest
+task :default => :test
```
Comment (obilodeau) [inline Rakefile]: We should probably add `test` to travis' script so that the tests run on PR, etc.

Comment (Mogztter) [reply Rakefile]: Good idea and done!

**Comment by Mogztter** [comment]
Thanks Olivier 🚀 🚀 🚀

**Pull request merged**